src/Centreon/Infrastructure/PlatformTopology/PlatformTopologyRepositoryRDB.php

```
@@ -209,4 +211,126 @@ public function findPlatform(int $serverId): ?Platform
 
         return $platform;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function deletePlatform(int $serverId): void
+    {
+        try {
+            $statement = $this->db->prepare(
+                $this->translateDbName('DELETE FROM `:db`.`platform_topology` WHERE id = :serverId')
+            );
+            $statement->bindValue(':serverId', $serverId, \PDO::PARAM_INT);
+            $statement->execute();
+        } catch (\Exception $ex) {
+            throw new RepositoryException(_('An error occured while deleting the Platform'));
```

@callapa , should we throw an exception here ?